### PR TITLE
Delete what to expect on teacher training blog post

### PR DESCRIPTION
**Pull request by MaxineCl**
### Trello card
https://trello.com/c/dEDUTjtR/2186-delete-and-redirect-blog-post-on-what-to-expect-on-your-teacher-training

### Context
We want to delete this blog post https://getintoteaching.education.gov.uk/blog/what-to-expect-on-your-teacher-training.
We have an information page on what to expect in teacher training https://getintoteaching.education.gov.uk/train-to-be-a-teacher/initial-teacher-training so we can add any additional information that’s useful from the blog post to this page.

### Changes proposed in this pull request

- Add any relevant content to the information page
- Delete blog post
- Set up blog post redirect

### CODE REVIEW

**Comment by MaxineCl** [comment]
@gemmadallmandfe I'm not sure most of the info in the blog post works for the page as it's specific to PGCE so I don't know whether it may confuse people. I've added in a section about holidays but happy to take it out for the reason I just said if you think it may confuse people.

File: app/views/content/train-to-be-a-teacher/initial-teacher-training.md [hunk 1]
```diff
@@ -79,6 +78,9 @@ School placements vary in length and structure, but the minimum length is usuall
 
 Read about [what teacher training is like doing a QTS with PGCE with one training provider](/blog/what-to-expect-on-your-teacher-training).
 
+## Holidays
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/initial-teacher-training.md:81]: I think you are right - this is confusing as we don't have the info for non-PGCE courses. Let's remove for now and look at enhancing the page properly under a future ticket

**Comment by gemmadallmandfe** [comment]
@MaxineCl There are further links to the blog post on the following pages, which is why you are getting build failures:

* /train-to-be-a-teacher/what-is-a-pgce
* /funding-and-support/salaried-teacher-training
* /funding-and-support/scholarships-and-bursaries

File: config/redirects.yml [hunk 2]
```diff
@@ -348,6 +348,7 @@ redirects:
   "/blog/9-tips-to-help-you-apply-for-teacher-training": "/how-to-apply-for-teacher-training"
   "/blog/choosing-the-right-teacher-training-course-provider": "/train-to-be-a-teacher/how-to-choose-your-teacher-training-course"
   "/blog/how-to-find-school-experience": "/is-teaching-right-for-me/get-school-experience"
+  "/blog/what-to-expect-on-your-teacher-training": " /train-to-be-a-teacher/initial-teacher-training"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:351]:
```suggestion
  "/blog/what-to-expect-on-your-teacher-training": "/train-to-be-a-teacher/initial-teacher-training"
```
@MaxineCl your redirect is currently throwing a 500 error - once you remove the extra space, it should clear up a build failure

**Comment by gemmadallmandfe** [comment]
@martyn-w Deleting this blog post is causing one of the spec tests to fail as we now only have 3 blogs under /blog/tag/advisers rather than 4

Could you update the spec test please? Thanks!

**Comment by martyn-w** [comment]
I've updated the blog spec test to reduce the count from 4 to 3. 

@MaxineCl there are a couple of other failing tests which require fixing:

* invalid path /blog/what-to-expect-on-your-teacher-training on page /funding-and-support/salaried-teacher-training
* invalid redirect: bad URI(is not URI?): "http://www.example.com/ /train-to-be-a-teacher/initial-teacher-training?abc=def&ghi=jkl"

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**